### DISCUSSION TITLE
v3.0.0-beta.9

### DISCUSSION THREAD

**Discussion by sserrata** [PaloAltoNetworks/docusaurus-openapi-docs#776] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/releases/tag/v3.0.0-beta.9'>v3.0.0-beta.9</a>.</em>

**Comment by alvinometric** [comment]
Great to see the progress on this 🙌
